script.js

```diff
@@ -0,0 +1,103 @@
+const localePixelBoard = document.getElementById('pixel-board');
+const localeColorPalette = document.getElementById('color-palette');
+const localeInputPixel = document.getElementById('board-size').value;
+const localeButtonBoard = document.getElementById('generate-board');
+const localeButtonClearBoard = document.getElementById('clear-board');
+let colorSelected = 'black';
+let sizePixel = 5;
+const colorList = [
+  'Blue ',
+  'Green',
+  'Red',
+  'Orange',
+  'Violet',
+  'Indigo',
+  'Yellow ',
+  'Gainsboro ',
+  'LightGray ',
+  'Silver ',
+  'DarkGray ',
+  'DimGray ',
+  'Gray ',
+  'SlateGray ',
+  'DarkSlateGray ',
+];
+
+function colorPalletteRondom() {
+  const localeColor = document.querySelectorAll('.color');
+  localeColor.forEach((element) => {
+    const colorRondm = Math.floor(Math.random() * 14);
+    if (element.className.includes('black')) {
+      element.style.backgroundColor = 'black';
+    } else {
+      element.style.backgroundColor = colorList[colorRondm];
+    }
+  });
```

Comment:
```suggestion
function colorPalleteRandom() {
  const localeColor = document.querySelectorAll('.color');
  localeColor.forEach((element) => {
    const colorRandom = Math.floor(Math.random() * 14);
    if (element.className.includes('black')) {
      element.style.backgroundColor = 'black';
    } else {
      element.style.backgroundColor = colorList[colorRandom];
    }
  });
```
Muito bom! Apenas o "a" e o "t" :100: